process-engine/bpmn-studio
# 🐛 :bug: Fix Loading of the Colorpicker

## What did you change?

The colorpicker still failed loading sometimes. 
This should now work reliably.

Closes #199.

## How can others test the changes?

- Open up BPMN-Studio
- See if the colorpicker is working
- Change the diagram
- See if the colorpicker is still working

## PR-Checklist

Please check the boxes in this list after submitting your PR:
